docs/spark-connector/spark-connector.md

```diff
@@ -27,11 +27,12 @@ The Gravitino Spark connector leverages the Spark DataSourceV2 interface to faci
 1. [Build](../how-to-build.md) or download the Gravitino spark connector jar, and place it to the classpath of Spark.
 2. Configure the Spark session to use the Gravitino spark connector.
 
-| Property                     | Type   | Default Value | Description                                                                                         | Required | Since Version |
-|------------------------------|--------|---------------|-----------------------------------------------------------------------------------------------------|----------|---------------|
-| spark.plugins                | string | (none)        | Gravitino spark plugin name, `com.datastrato.gravitino.spark.connector.plugin.GravitinoSparkPlugin` | Yes      | 0.5.0         |
-| spark.sql.gravitino.metalake | string | (none)        | The metalake name that spark connector used to request to Gravitino.                                | Yes      | 0.5.0         |
-| spark.sql.gravitino.uri      | string | (none)        | The uri of Gravitino server address.                                                                | Yes      | 0.5.0         |
+| Property                            | Type   | Default Value | Description                                                                                         | Required | Since Version |
+|-------------------------------------|--------|---------------|-----------------------------------------------------------------------------------------------------|----------|---------------|
+| spark.plugins                       | string | (none)        | Gravitino spark plugin name, `com.datastrato.gravitino.spark.connector.plugin.GravitinoSparkPlugin` | Yes      | 0.5.0         |
+| spark.sql.gravitino.metalake        | string | (none)        | The metalake name that spark connector used to request to Gravitino.                                | Yes      | 0.5.0         |
+| spark.sql.gravitino.uri             | string | (none)        | The uri of Gravitino server address.                                                                | Yes      | 0.5.0         |
+| spark.sql.gravitino.supportsIceberg | string | (none)        | Set to `true` to use Gravitino Iceberg catalog.                                                     | No       | 0.5.1         |
```

Comment:
if it is not required, it seems must have a default value.

Reply:
WDYT @jerryshao ?

Reply:
What's the meaning of `none`, it should either be false or true.

Reply:
ok